app/helpers/participants_helper.rb

```diff
@@ -6,4 +6,15 @@ def hall_of_fame_title
   def participant_badge_count(participant)
     participant.badge_acquisitions_count || 0
   end
+
+  def participant_image_profile_link(participant)
+    link_to edition_participant_path(@edition, participant).sub(/\d+\z/, participant.username) do
```

Comment:
@frmendes Did not really thought about that, only limited myself to move it to the helper file. I will review it and I'll get back to you ASAP :+1: